packages/report-server/src/reportBuilder/functions/utils.ts

```diff
@@ -28,3 +29,19 @@ export const dateStringToPeriod = (period: string, targetType: string): string =
   }
   return baseDateStringToPeriod(period, targetType);
 };
+
+export const fractionAndPercentage = (numerator: number, denominator: number) => {
+  const valueType = 'fractionAndPercentage';
+  if (typeof numerator !== 'number' || typeof denominator !== 'number') {
+    throw new Error(
+      `Function 'fractionAndPercentage' expect ${numerator} and ${denominator} as number`,
+    );
+  }
+  if (denominator === 0) {
+    throw new Error('One of the denominators is 0');
+  }
```

Comment:
I was thinking to prevent a case when we have a numerator >0 but a denominator 0, so that will give a bit inside about the data. 

With that, let's say in the future, data steward will know if there is one denominator is 0, instead of having results which eliminates this set of data.

Reply:
Yeah it's a good point... I've added it to the dev chat agenda so we can get some people's opinion on it? I think it'll be good to be consistent in the future! For now am totally happy with this 👍